### PR TITLE
Instructions to setup project added in readme.

### PR DESCRIPTION
**Pull request by hasalfernando**
Fixes #
#5 

## Proposed Changes

  - Added instructions on how to clone and run the project in a local environment
  - Added setting up the project to the table of contents.
  - Updated social media links to codeJunction links in the readme.
  - Removed CHANGELOG.md from file structure as it doesn't exist anymore.

## Screenshots

Screenshot (4)

Screenshot (5)

### CODE REVIEW

**Review by rajitha1998 — APPROVED** [review]

**Review by janethavi — CHANGES_REQUESTED** [review]
@hasalfernando Shall we rename the main header of the readme to codeJunction official website and remove the unwanted badges because all of the existing badges are working with the template source-code repo.
Also, can we remove the unwanted content? (versions, demo, quick start, documentation, resources, technical support and questions, useful links)
For the time being lets remove the file structure tree as well. The reason is, our file structure is getting changed very frequently as we are still doing the changes.
Reporting issues section needs to have the content of how to report an issue regarding our website

**Comment by hasalfernando** [comment]
> @hasalfernando Shall we rename the main header of the readme to codeJunction official website and remove the unwanted badges because all of the existing badges are working with the template source-code repo.
> Also, can we remove the unwanted content? (versions, demo, quick start, documentation, resources, technical support and questions, useful links)
> For the time being lets remove the file structure tree as well. The reason is, our file structure is getting changed very frequently as we are still doing the changes.
> Reporting issues section needs to have the content of how to report an issue regarding our website

Sure. Working on it.

**Comment by rajitha1998** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 48
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118&bid=17456105)

File: README.md [hunk 1]
```diff
@@ -1,236 +1,93 @@
...
+6. Next step is to fill out the template with accurate information about the bug or feature.
+
+Here are some advices for our users that want to report an issue:
+1. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
+2. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.
+3. If it is a new feature, try to explain it in a manner in which the reader would be able to get a fair understanding about it.
+
+## Licensing
+
+- Copyright 2018 Creative Tim (https://www.creative-tim.com)
```
Comment (rajitha1998) [inline README.md:79]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md [hunk 2]
```diff
@@ -1,236 +1,93 @@
...
+### Clone the project you forked
 
-If you have questions or need help integrating the product please [contact us](https://www.creative-tim.com/contact-us) instead of opening an issue.
+The URL of the project you cloned would look like 
+(Replace [your_username] with your github username.)
+``` https://github.com/[your_username]/codeJunction-site```
 
-## Licensing
+You can clone it to your local machine by typing and entering the below command in your terminal. 
+(Replace [your_username] with your github username.)
```
Comment (rajitha1998) [inline README.md:35]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md [hunk 3]
```diff
@@ -1,236 +1,93 @@
-# [Paper Kit 2 Angular - Free Bootstrap 4 UI Kit](https://demos.creative-tim.com/paper-kit-2-angular/)
+# [codeJunction Official Website](https://codejunctionteam.github.io/codeJunction-site)
 
-![version](https://img.shields.io/badge/version-1.3.0-blue.svg) [![license][license-badge]][LICENSE] [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/paper-kit-2-angular.svg?maxAge=2592000)](https://github.com/creativetimofficial/paper-kit-2-angular/issues?q=is%3Aopen+is%3Aissue) [![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/creativetimofficial/paper-kit-2-angular.svg?maxAge=259200)](https://github.com/creativetimofficial/paper-kit-2-angular/issues?q=is%3Aissue+is%3Aclosed) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
-![Paper Kit 2 Angular](https://s3.amazonaws.com/creativetim_bucket/products/65/original/opt_pk2_angular_thumbnail.jpg "Paper Kit 2 Angular Free")
+## Table of Contents
 
-Paper Kit 2 Angular is a free Bootstrap 4 UI Kit with pale colors, beautiful typography built for Angular 7.
+* [Setting Up the Project](#setting-up-the-project)
```
Comment (rajitha1998) [inline README.md:5]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md [hunk 4]
```diff
@@ -1,236 +1,93 @@
...
+
+Here are some advices for our users that want to report an issue:
+1. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
+2. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.
+3. If it is a new feature, try to explain it in a manner in which the reader would be able to get a fair understanding about it.
+
+## Licensing
+
+- Copyright 2018 Creative Tim (https://www.creative-tim.com)
+- Creative Tim [license](https://www.creative-tim.com/license)
```
Comment (rajitha1998) [inline README.md:80]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md (hunk 1 above)
Comment (rajitha1998) [inline README.md:79]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

Comment (rajitha1998) [inline README.md:79]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md (hunk 4 above)
Comment (rajitha1998) [inline README.md:80]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md [hunk 5]
```diff
@@ -1,236 +1,93 @@
...
 
-![version](https://img.shields.io/badge/version-1.3.0-blue.svg) [![license][license-badge]][LICENSE] [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/paper-kit-2-angular.svg?maxAge=2592000)](https://github.com/creativetimofficial/paper-kit-2-angular/issues?q=is%3Aopen+is%3Aissue) [![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/creativetimofficial/paper-kit-2-angular.svg?maxAge=259200)](https://github.com/creativetimofficial/paper-kit-2-angular/issues?q=is%3Aissue+is%3Aclosed) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
-![Paper Kit 2 Angular](https://s3.amazonaws.com/creativetim_bucket/products/65/original/opt_pk2_angular_thumbnail.jpg "Paper Kit 2 Angular Free")
+## Table of Contents
 
-Paper Kit 2 Angular is a free Bootstrap 4 UI Kit with pale colors, beautiful typography built for Angular 7.
+* [Setting Up the Project](#setting-up-the-project)
+* [Reporting Issues](#reporting-issues)
+* [Licensing](#licensing)
+* [Social Media](#social-media)
```
Comment (rajitha1998) [inline README.md:8]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

File: README.md (hunk 2 above)
Comment (rajitha1998) [inline README.md:35]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)

**Review by janethavi — APPROVED** [review]
Nice Work @hasalfernando

**Pull request merged**